packages/adfind/PKGBUILD

```diff
@@ -2,28 +2,31 @@
 # See COPYING for license details.
 
 pkgname=adfind
-pkgver=31.9808cb7
+pkgver=v1.0.0
 pkgrel=1
 pkgdesc='Simple admin panel finder for php,js,cgi,asp and aspx admin panels.'
 groups=('blackarch' 'blackarch-webapp' 'blackarch-recon')
 arch=('any')
 url='https://github.com/sahakkhotsanyan/adfind'
 license=('custom:unknown')
-depends=('php' 'curl' 'git')
-makedepends=('git')
+makedepends=('git' 'go')
 source=("git+https://github.com/sahakkhotsanyan/$pkgname.git")
 sha512sums=('SKIP')
 
 pkgver() {
   cd $pkgname
 
-  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+  echo $(git describe --abbrev=0 --tags)
```

Comment:
please use this instead:

https://github.com/BlackArch/blackarch-pkgbuilds/blob/581198db183f275d3d1f5b0135c94fb60df1b96a/PKGBUILD-generic-git#L24